src/features/welcome/welcome-page.js

```diff
@@ -1,17 +1,181 @@
 import { useNavigate } from 'react-router-dom';
-import Button from '@mui/material/Button';
-
 import { ROUTES } from 'common/constants';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import styles from './welcome-page.module.scss';
+import LogoSVG from '../../assets/img/logo.svg';
+import Logo from '../../components/logo/logo';
+import Feature from '../../components/Feature';
+import TeamMember from '../../components/TeamMember';
+import image1 from '../../assets/img/Billy-illust-01.png';
+import image2 from '../../assets/img/Billy-illust-02.png';
+import image3 from '../../assets/img/Billy-illust-03.png';
+import image4 from '../../assets/img/Billy-illust-04.png';
```

Comment:
Nothing wrong but ilustrations can have a more descriptive name so that you know what is in there without having to open the image.